blog/docs/articles/performance-test-with-slos.md

```diff
@@ -0,0 +1,183 @@
+# Dead Simple Performance Testing and SLOs for Knative Apps
+
+**Author: [Srinivasan Parthasarathy](https://researcher.watson.ibm.com/researcher/view.php?person=us-spartha), Senior Research Scientist and Manager, DevSecOps @ [IBM Research](https://research.ibm.com)**
+
+**Date: 2022-07-18**
+
+
+> Launch performance tests for Knative services (HTTP and gRPC) and validate their service-level objectives (SLOs) **in seconds**.
```

Comment:
Not sure if this is what you were looking for in terms of formatting...

<img width="504" alt="image" src="https://user-images.githubusercontent.com/44698588/179822259-6693251d-074a-4ee4-818c-a303f1d7d841.png">

Reply:
I removed the tagline. It is actually repeated throughout the article, so may be unnecessary :)